libraries/AP_Math/AP_Math.h

```diff
@@ -132,7 +132,7 @@ inline int32_t constrain_int32(const int32_t amt, const int32_t low, const int32
 }
 
 // degrees -> radians
-static inline float radians(float deg)
+static inline double radians(double deg)
```

Comment:
we should not change it here, as most flight controllers don't have double precision hw support
if there is an issue in SITL then it needs fixing in SITL